### PR TITLE
Instrumented Quote to work with Zipkin

### PR DESCRIPTION
**Pull request by AliceProxy**
Added tracing to quote. Requires setting environment variables to begin report tracing to Zipkin. Example service:
```
 ---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: quote
spec:
  replicas: 1
  selector:
    matchLabels:
      app: quote
  strategy:
    type: RollingUpdate
  template:
    metadata:
      labels:
        app: quote
    spec:
      containers:
      - name: backend
        image: alicewasko/quotezip:1.0
        ports:
        - name: http
          containerPort: 8080
        env:
        - name: ZIPKIN_SERVER
          value: "zipkin.default"
        - name: ZIPKIN_PORT
          value: "9411"
```

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -45,6 +49,9 @@ const (
 	EnvTLS         = "ENABLE_TLS"
 	EnvOpenAPIPath = "OPENAPI_PATH"
 	EnvRPS         = "RPS"
+	EnvZipkin	   = "ZIPKIN_SERVER"
```
Comment (nbkrause) [inline main.go:52]: NIT: Formatting here is weird

File: main.go [hunk 2]
```diff
@@ -342,6 +370,30 @@ func (s *Server) Logout(w http.ResponseWriter, r *http.Request) {
...
+		if err != nil {
+			log.Fatalln(err)
+		}
+
+		zipkinEndpoint := fmt.Sprintf("%s%s%s%d%s", "http://", zipkinServer, ":", zipkinPort, "/api/v2/spans")
+
+		tracer, err := buildTracer(zipkinEndpoint)
+		if err != nil {
+			log.Fatal(err)
+		} else {
```
Comment (nbkrause) [inline main.go:388]: NIT: `log.Fatal` will exit if called so there is no reason to put this in an `else` block

**Review by nbkrause — APPROVED** [review]
Incredible work! 

Some very small nit picks but this is great! Thanks!

File: main.go (hunk 1 above)
Comment (AliceProxy) [reply main.go:52]: It is weird, especially since I still don't see that formatting in vscode.

**Pull request merged**